crates/accelerate/src/equivalence.rs

```diff
@@ -664,6 +687,7 @@ impl EquivalenceLibrary {
         self.key_to_node_index.contains_key(&key)
     }
 
+    /// Returns an iterator with all the Keys in the `EquivalenceLibrary`.
```

Comment:
For Rust doc strings, we should use the Rust-style for code refs, e.g. `[EquivalenceLibrary]`.